src/__experimental__/components/simple-color-picker/color-sample-box/color-sample-box.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from 'styled-components';
+import StyledColorSampleBox from './style/color-sample-box.style';
+import baseTheme from '../../../../style/themes/base';
+import StyledTickIcon from '../tick-icon/tick-icon.style';
+import { THEMES } from '../../../../style/themes';
+
+const ColorSampleBox = ({ theme, color, checked }) => {
+  const tickIcon = checked && theme.name === THEMES.classic ? <StyledTickIcon checked type='tick' /> : null;
+  return <StyledColorSampleBox color={ color }>{tickIcon}</StyledColorSampleBox>;
+};
+
+ColorSampleBox.defaultProps = {
+  theme: baseTheme
+};
+
+ColorSampleBox.propTypes = {
+  theme: PropTypes.object,
```

Comment:
we don't need this prop here, you're pssing it through the `withTheme` HOC

Reply:
I removed baseTheme as default props but left theme in prop types checking to avoid eslint error about theme prop missing in prop validations